pkg/nuctl/command/deploy.go

```diff
@@ -191,3 +194,33 @@ func parseResourceAllocations(values stringSliceFlag, resources *v1.ResourceList
 
 	return nil
 }
+
+func parseVolumes(volumes stringSliceFlag) []functionconfig.Volume {
```

Comment:
Almost all functions should return an error, especially functions that get user input. If the user passes something invalid (e,g `foo`), it should return an error and the command should fail